test/db/cmd/cmd_wox

```diff
@@ -33,7 +33,7 @@ wox 0xabcdef@!10
 ps 14
 EOF
 EXPECT=<<EOF
-helloworld\x00\x00\x00\x00
+è\x83Ǣ\x98Ŀ\x83\xcf\x00\x00\x00\x00
```

Comment:
Looks like an unexpected result

Reply:
I think *not* having helloworld is definitely right (the test does write "helloworld" and then xor it with "0xabcd"), however there is a difference in how `stable`-rizin and `dev`-rizin prints the string...

```
$ rizin -qv
0.3.4
$ rizin =
[0x00000000]> wx c3a883c7a298c4bf83cf
[0x00000000]> ps 14
\xc3\xa8\x83\xc7\xa2\x98\xc4\xbf\x83\xcf\x00\x00\x00\x00
[0x00000000]> 
```

```
$ ./build-dev/binrz/rizin/rizin -qv
0.4.0-git
$ ./build-dev/binrz/rizin/rizin =
[0x00000000]> wx c3a883c7a298c4bf83cf
[0x00000000]> ps 14
è\x83Ǣ\x98Ŀ\x83\xcf\x00\x00\x00\x00
[0x00000000]> 
```

Reply:
that is due how ps was working before (i.e. not working)

Reply:
So I guess it is alright as it is within this new branch, right?

Reply:
yes, but i would print using px here, not ps